docs/WEBSOCKETS.md

```diff
@@ -0,0 +1,39 @@
+# WebSockets
+
+All apps and services in this archetecture use WebSockets to communicate between
+each other. In order for a easy level of interoperability, we've come up with a
+short set of standards.
+
+## Topics
+
+There are two forms of topics:
+
+### Events
+
+These are expressions of information - something has changed in the state of the
+programme.
```

Comment:
"programme" makes me think of a radio programme? So maybe onn Line 3 we need to quality that we'll call "apps and services (we'll call this programmes)"? Or find another word?

Reply:
"... in the state of your app or service"?